docs/reference/search/retriever.asciidoc

```diff
@@ -517,14 +551,137 @@ POST movies/_search
 This retriever uses a standard `match` query to search the `movie` index for films tagged with the genre "drama".
 It then re-ranks the results based on semantic similarity to the text in the `inference_text` parameter, using the model we uploaded to {es}.
 
+[[rule-retriever]]
+==== Query Rules Retriever
+
+The `rule` retriever offers users fine-grained control over the search results by applying contextual <<query-rules>> to pin or exclude documents for specific queries.
+This retriever performs similar functionality to the <<query-dsl-rule-query>> but works out of the box with other retrievers including reranking retrievers like <<text-similarity-reranker-retriever, text_similarity_reranker>> and <<rrf-retriever, rrf>>.
```

Comment:
```suggestion
This retriever has similar functionality to the <<query-dsl-rule-query>>, but works out of the box with other retrievers.
```

nit for brevity

Do we need `including reranking retrievers like <<text-similarity-reranker-retriever, text_similarity_reranker>> and <<rrf-retriever, rrf>>`?